### PR TITLE
S3 upload automation

### PR DESCRIPTION
**Pull request by jmervine**
# Context

After a discussion with @jdorfman, we'd like to leverage using Github as the source of truth for automated syncing of cdn files to S3. This is to ensure prompt release of new files, as well as remove any question about state.

# Solution

Leverage existing ops automation, from https://github.com/jmervine/bootstrapcdn-ops. Wrap it in a Docker image for execution and use CI to automate via Github Actions.

# Testing

1. Docker image should run if configured properly.
1. CI should automatically run and sync when this branch is deployed, either at merge time, or actual deployment time, depending on how CI is implemented.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I can't actually comment on how this works, but why don't we use an existent action?

**Comment by jmervine** [comment]
Not sure what "an existent action" is.

**Comment by XhmikosR** [comment]
https://github.com/marketplace?type=actions&query=s3 or another search should give more insight.

File: .github/workflows/updater.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Update S3
+
+on:
+  push:
+    branches:
+      - master
```
Comment (XhmikosR) [inline .github/workflows/updater.yml:6]: Should probably be develop? `master` is only for production, right?

Comment (jmervine) [reply .github/workflows/updater.yml:6]: Yes, this should only be run when deploying to production.

**Comment by jimaek** [comment]
I believe this is not necessary anymore, too bad because its a cool solution

**Pull request closed without merging**